sbom/examples/product/rhel-99.9.spdx.json

```diff
@@ -0,0 +1,65 @@
+{
+  "spdxVersion": "SPDX-2.3",
+  "dataLicense": "CC0-1.0",
+  "SPDXID": "SPDXRef-DOCUMENT-4d3d9002-fd75-4329-92f2-4271e186e32c",
+  "creationInfo": {
+    "created": "2006-08-14T02:34:56-06:00",
+    "creators": [
+      "Tool: example SPDX document only"
+    ]
+  },
+  "name": "RHEL 9.2 EUS",
+  "documentNamespace": "https://access.redhat.com/security/data/sbom/beta/spdx/rhel-9.2-eus-6a659023-7a52-4792-a66c-a7f19ec14ba8.json",
```

Comment:
To decide: I'm not sure what we should be naming these and what their location will be. Should they be named after the component being built, or the product?

Reply:
For `name` I think just the product is fine.
For `documentNamespace` it would ideally be unique to this SBOM, right? (If we ever want to use `externalDocumentRefs` in future.) So it will probably want to include the product name and some sort of hash of a sorted list of the component names and versions -- or even just a UUID.

Reply:
Thinking again, maybe `documentNamespace` isn't so important here -- these partial SBOMs are unlikely to be published in their own right, and so equally unlikely to be referenced externally.

Reply:
Ack on name, I'll make a note somewhere in the docs that it should be sourced from the RPA (combination of product_name+product_version).

I'll use a generic URL for `documentNamespace` so that it's clear it's not actually accessible at that URL, and a stream ID + UUID combo.